plugins/web/ui/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Build and install the frontend as a static site.
+
+file(GLOB_RECURSE ui-sources *.svelte *.ts)
+
+#find_program(pnpm)
+
+set(www_dir "${VAST_CMAKE_INSTALL_DATADIR}/vast/plugin/web/www")
+
+add_custom_command(
+  OUTPUT "${PROJECT_BINARY_DIR}/${www_dir}"
+  WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+  COMMAND pnpm install
+  COMMAND pnpm build
+  COMMAND
+    "${CMAKE_COMMAND}" -E copy "${CMAKE_CURRENT_SOURCE_DIR}/build" "${PROJECT_BINARY_DIR}/${www_dir}"
```

Comment:
You probably don't wanna build this in the source directory, but rather copy the sources over to the build directory beforehand.

Reply:
Why?

Reply:
Because `rm -rf path/to/build` should clean up everything that `cmake -B path/to/build && cmake --build path/to/build` produced. Currently this is producing build artifacts in the source directory.

Reply:
This is resolved now.